packages/react-components/src/components/RichTextEditor/RichTextEditor.tsx

```diff
@@ -202,6 +204,7 @@ export const RichTextEditor = React.forwardRef<RichTextEditorComponentRef, RichT
           data-testid={'rooster-rich-text-editor'}
           // if we don't use 'allowKeyboardEventPropagation' only the enter key is caught
           onKeyDown={props.onKeyDown}
+          focusOnInit={autoFocus === 'sendBoxTextField'}
```

Comment:
This fix might not work very well for situations when we already have inital text for editor (edit messasge scenario). Please see comments in `editorCreator` for more details

Reply:
Just to share a little bit more info, currently focus will be set to editor all the time because of the implementation of `editorCreator`

Reply:
> Just to share a little bit more info, currently focus will be set to editor all the time because of the implementation of `editorCreator`

Hi @vhuseinova-msft, tested with Chat composite and it works as expected. The autoFocus API's goal is to set the focus when the page first loaded and it's exactly what the focusOnInit API from RoosterJS is doing.

https://github.com/Azure/communication-ui-library/assets/107075081/a1b72d2f-8cb9-46e3-9df3-8c0a4896538b

Reply:
> This fix might not work very well for situations when we already have inital text for editor (edit messasge scenario). Please see comments in `editorCreator` for more details

Edit message scenario does not need this API since edit box does not exist at the time when the page just loaded. For our current implementation with Sendbox and MessageThread, this autoFocus API was not added to the MessageThread component to pass down to the edit box.